WalletWasabi.Fluent/ViewModels/StatusIcon/StatusIconDesignViewModel.cs

```diff
@@ -2,6 +2,7 @@
 using System.Windows.Input;
 using ReactiveUI;
 using WalletWasabi.Fluent.AppServices.Tor;
+using WalletWasabi.Services.Tor.StatusChecker;
```

Comment:
It shouldn't! this is just a stub to get design-time data in the Avalonia Previewer. Did you see those items being displayed? 🤔

Reply:
Btw do these viewmodels really have that much advantage? Last time someone tried to modify an interface property nullability because a "design" viewmodel had a `null` set.

Reply:
Yes, it can be a really time saver when it comes to visual design. Without it, some views and controls show blank or impossible invalid states that are not useful to have a look of the real control. This is even more useful for controls that are rendered in a flyout, because inspecting those with the DevTools is impossible. Moreover, the Design.DataContext property is exactly for this kind of use cases :) 

The problem you mention should be a matter of deciding the correct nullability. Sometimes it's easy to set something to null when you don't mind about the actual value. We just need to be careful.